src/libraries/Common/tests/System/Net/Http/LoopbackServer.cs

```diff
@@ -379,6 +379,7 @@ public class Options : GenericLoopbackOptions
             public string Domain { get; set; }
             public string Password { get; set; }
             public bool IsProxy { get; set; } = false;
+            public X509Certificate2 Certificate { get; set; }
```

Comment:
Would it make sense to put the certificate in `GenericLoopbackOptions` and make use if it in other loopback servers?

Reply:
I like that idea - to keep all ssl together even if we don't need it right now. updated.